### DISCUSSION TITLE
Welcome to Frictionless Data discussions!

### DISCUSSION THREAD

**Discussion by lwinfree** [frictionlessdata/frictionlessdata.io#667] [General]
Welcome! This is a space to discuss big-picture questions about the Frictionless Data project. 

Please open a new discussion if you have a new topic you'd like to discuss, and read through the existing discussions to let us know your thoughts.

Please read our [Code of Conduct](https://github.com/frictionlessdata/.github/blob/d29de1233604c4a242974c68d37f706f69e7b14b/CODE_OF_CONDUCT.md). By participating on this site, you are agreeing to our Code of Conduct.

Do you have a bug report or question that is specific to a particular repository? Please open an issue in that repository instead of a Discussion. Unsure if that applies to you? Don't worry - please open a discussion here & we can move it to the right place 😃 

Need more help? Chat with us on our [Discord community chat](https://discord.com/invite/j9DNFNw)! 

Thank you for participating in the Frictionless Data project!

**Comment by lwinfree** [comment]
Also, if you need help creating a discussion or participating in a discussion post, please comment here!

**Comment by loleg** [comment]
I found this point rather salient in a [comparison of popular platforms](https://orbit.love/blog/slack-vs-discord-vs-discourse):

> Communities focused on collaboration usually want to foster deeper participation from existing community members, lower the bar for new folks to get involved, solve problems as a group, and publicly share the solutions. Discourse stands out in each of these areas. Since all content is public and indexed by search engines, members can reference previously answered questions and discussions, which means the core team can avoid answering the same questions repeatedly. New members can quickly learn the basics and overcome common challenges, and existing members can pose more complex questions and receive help from the broader community.

I believe these important points mostly apply to GitHub Discussions as well. The terms, [closed](https://github.com/github/feedback/discussions/43) [API](https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions), and proprietary nature here do leave something to be desired. Nevertheless I am a huge fan of :octocat: octocats, and hope this new hub for the Frictionless Data community will thrive!

For anyone else who wishes they could see new discussion threads or responses elsewhere: it sounds like [webhook support](https://github.com/integrations/slack/issues/1026) will enable subscribing to Discussions on other platforms, which would be very helpful.. (`<ahem>`[RSS, anyone?](https://github.com/github/feedback/discussions/31)`</ahem>`)
